### PR TITLE
Task/1-11

### PR DESCRIPTION
**Pull request by mcksyrek**
Tasks from 1 to 11 resolved

### CODE REVIEW

**Comment by Tsvetkovv** [comment]
PR is hard readable due to mix of style changes and business logic. I see a few options:
1. Create separate PR to apply your code style settings for whole app (tasks/tests) at once.
2. Adjust your code style rules to match with existing code
3. Don't commit or add to ignore code style fixes for specs

File: tasks/3_is-boolean/is-boolean.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export const isBoolean = (argument) => {
+  return typeof argument == 'boolean';
```
Comment (Tsvetkovv) [inline tasks/3_is-boolean/is-boolean.js]: it's better to always use triple equals:
https://eslint.org/docs/rules/eqeqeq

File: tasks/5_profile-lookup/profile-lookup.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  ];
+
+  const foundContact = contactsList.find((person) => person.name === firstName);
+
+  if (foundContact) {
+    if (foundContact[prop]) return foundContact[prop];
+    else return 'No such property';
+  } else return 'No such contact';
+};
+
```
Comment (Tsvetkovv) [inline tasks/5_profile-lookup/profile-lookup.js]: If you set `address` for Akira as empty:
```js
    {
      name: 'Akira',
      address: '',
    },
```
tests would pass, but should not. Could you fix that?

Comment (mcksyrek) [reply tasks/5_profile-lookup/profile-lookup.js]: Last commit corrected this issue

File: tasks/5_profile-lookup/profile-lookup.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    },
+    {
+      name: 'Akira',
+    },
+  ];
+
+  let foundContact = contactsList.find((person) => person.name === firstName);
+
+  if (foundContact) {
+    let hasProp = Object.keys(foundContact).find((key) => key === prop);
```
Comment (Tsvetkovv) [inline tasks/5_profile-lookup/profile-lookup.js]: [operator In](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/in) :)

File: tasks/5_profile-lookup/profile-lookup.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    {
+      name: 'Harry',
+      likes: ['Intriguing Cases', 'Violin'],
+    },
+    {
+      name: 'Akira',
+    },
+  ];
+
+  let foundContact = contactsList.find((person) => person.name === firstName);
```
Comment (Tsvetkovv) [inline tasks/5_profile-lookup/profile-lookup.js]: const?

File: tasks/10_remove-whitespaces/remove-whitespaces.js [hunk 5]
```diff
@@ -0,0 +1,3 @@
+export const trimWhiteSpaces = (string) => {
+  return string.split(' ').join('');
```
Comment (ArtemLunt) [inline tasks/10_remove-whitespaces/remove-whitespaces.js]: If we have a short single line arrow function, we can make it shorter.
For example,
```
export const func = param => funcWhichReturnsRequiredResult();
```

File: tasks/11_find-longest-word/find-longest-word.js [hunk 6]
```diff
@@ -0,0 +1,5 @@
+export const findLongestWord = (string) => {
+  return string.split(' ').reduce((longest, current) => {
+    return longest > current.length ? longest : current.length;
```
Comment (ArtemLunt) [inline tasks/11_find-longest-word/find-longest-word.js]: Same thing.)

File: tasks/5_profile-lookup/profile-lookup.js (hunk 4 above)
Comment (ArtemLunt) [reply tasks/5_profile-lookup/profile-lookup.js]: We can make it shorter using destructuring.)

File: tasks/6_reverse-string/reverse-string.js [hunk 7]
```diff
@@ -0,0 +1,3 @@
+export const reverseString = (string) => {
+  return string.split('').reverse().join('');
```
Comment (ArtemLunt) [inline tasks/6_reverse-string/reverse-string.js]: Same about shorter version.

Comment (Tsvetkovv) [reply tasks/6_reverse-string/reverse-string.js]: I would like to leave as is  😋 long lines not such readable in my opinion

Comment (ArtemLunt) [reply tasks/6_reverse-string/reverse-string.js]: In this case we have not really long string.
I specified to @mcksyrek that this is just an optional suggestion.)

**Comment by ArtemLunt** [comment]
@mcksyrek, please, delete branches after merge.

**Pull request merged**